test/problems/fltcmp/input_validators/validate.py

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python3
+
+import re
+import sys
+
+line = sys.stdin.readline()
+if not re.match("([1-9][0-9]*)\n", line):
+    print("Positive integer expected", file=sys.stderr)
+    sys.exit(43)
```

Comment:
something is weird here. Either the sys.exit should go or the code directly below. I suspect the below should not be indented?

Reply:
fixed, thank you